liferay-tango/liferay-portal
# LPS-178780 Revert front-end code for the View part of the Segment Edit

Motivation
=======
It has been decided to stop segmentation unification, so for that, as a tango team we have decided to revert the code from master and move it to the branch with name LPS-154036-unified-segmentation-experience. The reason to do that, is to prevent problems in the future. The code involved in this PR is related to the epic and is the front-end of the View part in the Segments Editor.

[Link to technical task](https://issues.liferay.com/browse/LPS-178074)
[Link to technical subtask](https://issues.liferay.com/browse/LPS-178780)

Proposed Solution
========
1. Revert all code from master below FF related to the epic: https://issues.liferay.com/browse/LPS-166954

Steps to Verify:
----------------
1. Keep the FF LPS-166954
2. Go to Control Panel -> Segments
3. Create various new Segments of different contributors (User Segment, Organization Segment, Session Segment..) and of different types (Dates, String, Boolean..) and complete their necessary fields.
4. Save the Segment/s, re-open it and check that the values are correct and it doesn't throw any error in the View mode.
